### ISSUE TITLE
Test failure :Loader/classloader/generics/Layout/General/Base01b_seq/Base01b_seq.sh

### ISSUE THREAD

**Issue by AriNuer** [dotnet/runtime#33143]
**Job:**
[runtime-coreclr gcstress0x3-gcstress0xc:#20200302.1](https://dev.azure.com/dnceng/public/_build/results?buildId=543557&view=results)

**Error message:**
`
\nReturn code: 1\nRaw output file: /root/helix/work/workitem/Loader/classloader/Reports/Loader.classloader/generics/Layout/General/Base01b_seq/Base01b_seq.output.txt\nRaw output:\nBEGIN EXECUTION\n/root/helix/work/correlation/corerun Base01b_seq.dll ''\nTest Failed at location: 5\nTest Failed at location: 6\nTest Failed\nExpected: 100\nActual: 1\nEND EXECUTION - FAILED\nTest Harness Exitcode is : 1\nTo run the test:\n> set CORE_ROOT=/root/helix/work/correlation\n> /root/helix/work/workitem/Loader/classloader/generics/Layout/General/Base01b_seq/Base01b_seq.sh\nExpected: True\nActual: False
`

**Stack trace:**
`
at Loader_classloader._generics_Layout_General_Base01b_seq_Base01b_seq_._generics_Layout_General_Base01b_seq_Base01b_seq_sh() in /__w/11/s/artifacts/tests/coreclr/Linux.arm64.Checked/TestWrappers/Loader.classloader/Loader.classloader.XUnitWrapper.cs:line 10616
`

**Details:**
https://dev.azure.com/dnceng/public/_build/results?buildId=543557&view=ms.vss-test-web.build-test-results-tab&runId=17215806&resultId=101439&paneView=debug

**Comment by AriNuer** [comment]
Test `Loader/classloader/generics/Layout/General/Base01b_seq/Base01b_seq.sh` failed again here [runtime-coreclr gcstress0x3-gcstress0xc:20200405.1](https://dev.azure.com/dnceng/public/_build/results?buildId=589459&view=results)
**Error message:**
```
Return code: 1
Raw output file: /home/helixbot/work/A259091B/w/B072094F/e/Loader/classloader/Reports/Loader.classloader/generics/Layout/General/Base01b_seq/Base01b_seq.output.txt
Raw output:
BEGIN EXECUTION\n/home/helixbot/work/A259091B/p/corerun Base01b_seq.dll ''
Test Failed at location: 5
Test Failed at location: 6
Test Failed
Expected: 100
Actual: 1
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:
> set CORE_ROOT=/home/helixbot/work/A259091B/p
> /home/helixbot/work/A259091B/w/B072094F/e/Loader/classloader/generics/Layout/General/Base01b_seq/Base01b_seq.sh
Expected: True
Actual: False
```
**Stack trace:**
```
   at Loader_classloader._generics_Layout_General_Base01b_seq_Base01b_seq_._generics_Layout_General_Base01b_seq_Base01b_seq_sh() in /__w/1/s/artifacts/tests/coreclr/Linux.x64.Checked/TestWrappers/Loader.classloader/Loader.classloader.XUnitWrapper.cs:line 10616
```
**Details:**
https://dev.azure.com/dnceng/public/_build/results?buildId=589459&view=ms.vss-test-web.build-test-results-tab&runId=18580496&resultId=100362&paneView=debug

**Comment by AndyAyersMS** [comment]
This is likely a dup of #33366 which was fixed by #34613 just after the failure above.

No new failures in the past 30 days, so will close.
